{"name": "Проверка подтверждения заказа, передан неверный id курьера, запрос возвращает ошибку", "status": "passed", "start": 1747402270141, "stop": 1747402270355, "uuid": "5a734805-b3ee-40ed-a6f0-592e2f960ddd", "historyId": "4bdf989477d9e835d16acb4bd203da1e", "testCaseId": "4bdf989477d9e835d16acb4bd203da1e", "fullName": "tests.orders.test_accept_order.TestAcceptOrder#test_accept_order_wrong_courier_id", "labels": [{"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "MacBook-Pro-Mikhail.local"}, {"name": "thread", "value": "32108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_accept_order"}]}